### PR TITLE
Make tools/ac-client more automation friendly

### PR DESCRIPTION
**Pull request by MattHardcastle**
I'm working on a system to monitor my domains availability. The ac-client tool seems like the perfect for the job, but it's output is via Qt log messages, which are some what hard to parse. These changes add an -x and -f option to ac-client that turn off QT logging and enable a "full-output" mode. Used together they make parsing the output a little easier and don't impact the current functioning of the tool when unused.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10143/HighFidelity-Beta-PR13245-5da5c29d45bfcd67b79045c8b4030e56da4fba99.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10143/HighFidelity-Beta-PR13245-5da5c29d45bfcd67b79045c8b4030e56da4fba99.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10143/HighFidelity-Beta-Interface-PR13245-5da5c29d45bfcd67b79045c8b4030e56da4fba99.dmg

**Comment by MattHardcastle** [comment]
Thanks for the review @Atlante45. Very informative. It helped convinced me I need to finish [The C++ Programming Language](https://www.amazon.com/C-Programming-Language-4th/dp/0321563840) before attempting to inflict my non-idiomatic C++ code on the world. Already in the last week of readying I've learned about different possible approaches I would rather take. I can get what I need for now with my own side build of ac-client. In a couple weeks I will circle back and send you guys something that's, hopefully, a bit more polished than this PR. Thanks again.
